Public/Sdk/Public/Prelude/Prelude.OfficeBackCompatHacks.dsc

```diff
@@ -132,6 +132,8 @@ namespace Transformer {
         isLight?: boolean;
         /** Obsolete: You must use import {Transformer} from "Sdk.Transformers" */
         keepOutputsWritable?: boolean;
+        /** Obsolete: You must use import {Transformer} from "Sdk.Transformers" */
+        priority?: number;

```

Comment:
> priority?: number; [](start = 7, length = 19)

Since you need ot update codegen to use it. Would you mind removing this and adding `import {Transformer} from "Sdk.Transformers"` at the top of the generated file?

Reply:
Which file do you mean is generated?  Are you saying somewhere in Office code we should add this?